src/runtime/module_importer.js

```diff
@@ -12,11 +12,9 @@ exports.initTracer = function () {
     // the version provided by the layer
     const path = require.resolve("dd-trace", { paths: ["/var/task/node_modules", ...module.paths] });
     // tslint:disable-next-line:no-var-requires
-    const tracer = require(path).init({
-        tags: {
-            "_dd.origin": "lambda",
-        },
-    });
+    // add lambda tags to DD_TAGS environment variable
+    updateDDTags("_dd.origin:lambda")
+    const tracer = require(path).init();
```

Comment:
I'm not sure this is the right approach. I'd oppose in updating the env var. At the same time, I think it leaves our team with no room to add additional config if required.

Reply:
I've changed this to simply include the env variable values in the tags config arg